### ISSUE TITLE
Can't run wpilibutility after fresh install on Linux

### ISSUE THREAD

**Issue by spacey-sooty** [wpilibsuite/WPILibInstaller-Avalonia#414]
I get this error
```
[43180:0908/164239.614630:FATAL:setuid_sandbox_host.cc(158)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /home/jade/wpilib/2024/utility/chrome-sandbox is owned by root and has mode 4755.
Trace/breakpoint trap (core dumped)
```
These commands fix it
```
sudo chown root:root /home/jade/wpilib/2024/utility/chrome-sandbox
sudo chmod 4755 /home/jade/wpilib/2024/utility/chrome-sandbox
```

**Comment by rzblue** [comment]
Same as #411- we should be able to apply the `--no-sandbox` fix 
https://www.electronjs.org/docs/latest/api/command-line-switches#--no-sandbox

**Comment by sciencewhiz** [comment]
What OS?

**Comment by sciencewhiz** [comment]
> Same as #411- we should be able to apply the `--no-sandbox` fix 

we'd have to make a script for the utility since one doesn't exist right now, but it could be patterned after #412 

my guess is that AdvantageScope will have the same problem.

**Comment by spacey-sooty** [comment]
> What distro and version? Basically trying to figure out if this is only an Ubuntu 24.04 problem or if other distros have done the same thing to help with docs.

Im on Ubuntu 24.04

**Comment by sciencewhiz** [comment]
Closing as fixed in #419 with the installation instructions at https://docs.wpilib.org/en/latest/docs/zero-to-robot/step-2/wpilib-setup.html
